website/docs/react-native/santoku/design/demo/push-notification/overview.mdx

```diff
@@ -0,0 +1,96 @@
+# プッシュ通知
+
+## 概要
+
+プッシュ通知の機能を検証するデモページです。
```

Comment:
I feel we need a diagram to organize how users receive push notifications....
It is very complicated and difficult...
(can be anothor task)

Reply:
Added!
https://dev.azure.com/ws-4020/develop-mobile-app-casually/_backlogs/backlog/Mobile%20Dev%20Team/Epics/?workitem=16375

Reply:
[test]
AB#16375